packages/ubuntu_desktop_installer/lib/utils/version_extractor.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+
+class VersionExtractor {
+  static VersionExtractor? _instance;
+
+  static VersionExtractor get instance {
+    if (_instance == null) {
+      _instance = VersionExtractor._();
+    }
+    return _instance!;
+  }
+
+  VersionExtractor._();
+
+  String loadedVersion = '';
+
+  String _dummy() {
+    return 'Ubuntu';
+  }
+
+  String extractLocalVersion(String content) {
+    return content.substring(content.indexOf("\"") + 1, content.length - 1);
+  }
+
+  String getVersion({
```

Comment:
In line with my previous suggestion, I would rename that method to something like `getProductInfo()`.